monitor/frontend-src/src/components/Timer/index.js

```diff
@@ -2,13 +2,15 @@ import React from 'react'
 import { connect } from 'react-redux'
 import leftpad from 'left-pad'
 
+import './utils/moment'
```

Comment:
Use `yarn add moment` And then `import moment from 'moment'` instead, so it’s a dependency rather than a file (so we ie can update it).